### ISSUE TITLE
Multiple codetour panels pop up following recent change to VSCode

### ISSUE THREAD

**Issue by kazemicode** [microsoft/codetour#301]
Not sure of root cause yet. Video: https://www.youtube.com/watch?v=24jOFwE89yI

**Comment by jmussman** [comment]
Hey S! I was verifying issues on both Windows and MacOS, and I was typing up a message on Slack to see if you had seen the same when I saw this here. Apparently I'm a month late to the party but I want to add a little:

1. If you edit a step, three copies of the editing panel open up. You can edit in any of the three, but you need to stick to one..
2. There isn't any way to cancel out of the editor; if you try to close a panel with out making any changes the other panels stay open. If you close all of them in this state, you are still left in the edit view. Sometimes the save buttons are all disabled until you actually change anything, so the only way to back out is to close the tour.
3. If you change the description in one and close another, that doesn't work. If you close the one you made the changes in, that works.
4. The Add Step button disappeared. In fact, there isn't any way to add a step without manually editing the tour file.
5. The add step link in the CodeTour only shows up for the first step after you create a new empty tour.

**Comment by kazemicode** [comment]
Yup! Can confirm I have seen the same behavior. Sent from my iPhoneOn Sep 28, 2024, at 10:45 AM, Joel Mussman ***@***.***> wrote:﻿
Hey S! I was verifying issues on both Windows and MacOS, and I was typing up a message on Slack to see if you had seen the same when I saw this here. Apparently I'm a month late to the party but I want to add a little:

If you edit a step, three copies of the editing panel open up. You can edit in any of the three, but you need to stick to one..
There isn't any way to cancel out of the editor; if you try to close a panel with out making any changes the other panels stay open. If you close all of them in this state, you are still left in the edit view. Sometimes the save buttons are all disabled until you actually change anything, so the only way to back out is to close the tour.
If you change the description in one and close another, that doesn't work. If you close the one you made the changes in, that works.
The Add Step button disappeared. In fact, there isn't any way to add a step without manually editing the tour file.
The add step link in the CodeTour only shows up for the first step after you create a new empty tour.

**Comment by TMA-2** [comment]
I was about to open an issue for this, but it looks like it's already been done. So, I'll just add my own versions and a screenshot here.

## Versions
- OS: Windows 10 Enterprise `22H2`
- VSCode: `1.96.4`
- CodeTours: `0.0.59`

## Screenshots
![CodeTour - Replicated Edit Steps](https://github.com/user-attachments/assets/e6733aab-3fe6-4918-a746-9a6d38cb8704)

**Comment by lostintangent** [comment]
I believe this was a regression with VS Code, that's now been fixed. I'm not able to repro it anymore, so I'm going to close it for now. But please re-open this if you're still encountering this bug. Thanks!
